proto/mint/v1beta1/mint.proto

```diff
@@ -26,23 +27,23 @@ message DistributionProportions {
     (gogoproto.moretags) = "yaml:\"staking\"",
     (gogoproto.nullable) = false
   ];
-  // pool_incentives defines the proportion of the minted minted_denom that is
-  // to be allocated as pool incentives.
-  string pool_incentives = 2 [
+  // community_pool defines the proportion of the minted mint_denom that is
```

Comment:
should this comment say "community_pool_growth defines..."?

Reply:
done